### PR TITLE
Landing page setup

### PR DESCRIPTION
**Pull request by addybeuch**
## Description
Updated Landing Page design and navigation bar. 

contributing members:
Emmanuel Gatica @mannig1224
Daniel Kish @Daniehk832 
Emmanuel Gatica egatica51@gmail.com @mannig1224
Vicki Lei ylei1088@gmail.com @ylei1088 
Adam Beuchert addybeuch@gmail.com @addybeuch

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes
https://www.loom.com/share/7af15da2a30f4c9f8d413cac47007efb

### CODE REVIEW

**Review by riqthedev — COMMENTED** [review]
The landing page is looking really great so far! , A few adjustments to the proportion of the elements, remove horizontal scroll, and style corrections and you're good to go.

**Review by Danielhk832 — APPROVED** [review]
This looks like an awesome first step. I can tell a bunch of work went into this and I think it is super important that users get a good feel for what the company is about. great work!

File: src/components/pages/NavBarLanding/NavBarLanding.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import { Menu, Button, Layout, Breadcrumb } from 'antd';
+import './NavBarLanding.css';
+
+const { Header } = Layout;
+
+export default function NavBarLanding() {
```
Comment (ylei1088) [inline src/components/pages/NavBarLanding/NavBarLanding.js:8]: Good job on building the layout of the landing page, will continue refactor and add more code for the header and footer.

**Review by ylei1088 — APPROVED** [review]

**Review by riqthedev — APPROVED** [review]

**Review by krueng — APPROVED** [review]
Looks nice! Styling is responsive and the landing page tells a lot about the Underdog Devs in a few short sentences. Thanks for doing this!

**Pull request merged**